src/main/java/me/botsko/prism/wands/Wand.java

```diff
@@ -8,66 +8,26 @@
 
 public interface Wand {
 
-	/**
-	 * 
-	 */
-	void playerLeftClick(Player player, Location loc);
+    void playerLeftClick(Player player, Location loc);
 
-	/**
-	 * 
-	 */
-	void playerRightClick(Player player, Location loc);
+    void playerRightClick(Player player, Location loc);
 
-	/**
-	 * 
-	 * @param player
-	 * @param entity
-	 */
-	void playerRightClick(Player player, Entity entity);
+    void playerRightClick(Player player, Entity entity);
+    void setItemWasGiven(boolean given);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['METHOD_DEF' should be separated from previous statement.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)